### ISSUE TITLE
FP: Duplicate values in CASE statement (when utf-8)

### ISSUE THREAD

**Issue by stefandrissen** [Riverside-Software/sonar-openedge#1091]
Our cpstream / cpinternal is utf-8:
````
DEFINE VARIABLE cc AS CHARACTER NO-UNDO.

cc = 'Ï':u.

CASE cc:
   WHEN 'Í':u THEN MESSAGE 1.
   WHEN 'Ï':u THEN MESSAGE 2.
END CASE.
````

False positive with incorrect message, indicating that there are some codepage shenannigans going on:

![image](https://github.com/Riverside-Software/sonar-openedge/assets/5297464/40266bb1-6f03-49e7-8846-c12a7cd564a3)

**Comment by stefandrissen** [comment]
This seems somehow related to how PDSOE compiles a project, resources are defined as utf-8.

When ``pct:compile`` is used (with ``cpstream and ``cpinternal`` set to ``utf-8``)  the r-code is fine and displays the correct character.

When PDSOE is used, even when adding ``-cpstream utf-8`` and ``-cpstream utf-8`` to the project startup settings the correct case branch is excecuted, but the characters are still shown incorrectly.

If I change ``startup.pf`` to set cpinternal and cpstream to utf-8 then the characters are shown correctly, but SonarLint still complains.

**Comment by gquerret** [comment]
@stefandrissen Did you remove a comment ? I have an email from GitHub with an extra comment, but I don't see it here. Maybe GitHub issues this morning...

**Comment by stefandrissen** [comment]
> @stefandrissen Did you remove a comment ?
Yes, it was me irrelevantly unraveling some settings in my environment which were not relevant for the case.
